worldengine/drawing_functions.py

```diff
@@ -142,7 +140,7 @@ def _find_mountains_mask(world, factor):
                                            radius=3,
                                            predicate=world.is_mountain))
                 if v > 32:
-                    _mask[y][x] = v / 4
+                    _mask[y, x] = v / 4.0  # force conversion to float, Python 2 will *not* do it automatically
```

Comment:
Not sure which behaviour was the intended one here. The ancient map output looks different, I think the mountains are spread a little thinner for the old behaviour ```int(v/4)```.